docs/antctl.md

```diff
@@ -52,3 +52,199 @@ or Agent.
 When running out-of-cluster ("controller mode" only), antctl will look for your
 kubeconfig file at `$HOME/.kube/config` by default. You can select a different
 one with `--kubeconfig`.
+
+The following sub-sections introduce a few commands which are useful for
+troubleshooting the Antrea system.
+
+### `controllerinfo` and `agentinfo` commands
+`antctl` controller command `get controllerinfo` (or `get ci`) and agent command
+`get agentinfo` (or `get ai`) print runtime information of `antrea-controller`
+and `antrea-agent` respectively.
+```
+antctl get controllerinfo
+antctl get agentinfo
+```
+
+### NetworkPolicy commands
+Both Antrea Controller and Agent support querying NetworkPolicy objects.
```

Comment:
do you want to repeat here that for the Agent it is only local information (I make a generic statement about this at the beginning of the doc)?

Reply:
If you think ok, I would not repeat it here (otherwise the sentence will be much longer than the current version).

Reply:
sounds good